Pipfile

```diff
@@ -0,0 +1,38 @@
+[[source]]
+
+name = "pypi"
+verify_ssl = true
+url = "https://pypi.python.org/simple"
+
+[requires]
+python_version = "3.5"
+
+
+[packages]
+
+flask = "==0.12"
+flask-restful = "==0.3.5"
+flask-sqlalchemy = "==2.1"
+flask-cors = "==3.0.2"
+marshmallow = "==2.13.1"
+"psycopg2" = "==2.6.2"
+sqlalchemy = "==1.1.5"
+python-jose = "==1.2.0"
+structlog = "==16.1.0"
+cryptography = "==2.1.2"
+notifications-python-client = "==4.2.0"
+cfenv = "==0.5.3"
+gunicorn = "==19.7.1"
+
+
+[dev-packages]
+
+pylint = "*"
+"flake8" = "*"
```

Comment:
Playing devils advocate, if we are not enforcing flake8 in this repo do we need the import?

Reply:
Good point. We probably should be. I'll run it and see if it gives me any gyp

Reply:
We've got some error so going to leave it out for now. Will raise a tech debt ticket

Reply:
Yeh, I ran a flake8 check in tox and it wasn't pretty. Do we have anyway to lint the code?

Reply:
I guess add it in and fix the linting issues. I'll pick up after the pipenv stuff if I can

Reply:
NB: It would be useful to fail travis builds on linting issues as well.